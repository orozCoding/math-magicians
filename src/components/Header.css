a,
a::after,
a::before,
a:visited {
  text-decoration: inherit;
}

#header {
  align-self: center;
  width: 100%;
  justify-content: space-between;
  padding: 0 30px;
}

.logo {
  font-family: 'Bebas Neue', cursive;
  font-size: 60px;
}

ul {
  list-style: none;
}

.navBar ul {
  gap: 10px;
}

.activeLink {
  color: red;
}
