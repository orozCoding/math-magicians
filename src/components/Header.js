import React from 'react';
import Navbar from './Navbar';

const Header = () => (
  <header id="header" className="d-flex">
    <div className="logo-container d-flex">
      <p className="logo">Math Magicians</p>
    </div>
    <div>
      <Navbar />
    </div>
  </header>
);

export default Header;
