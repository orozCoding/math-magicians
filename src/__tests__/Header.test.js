import React from 'react';
import '@testing-library/react';
import renderer from 'react-test-renderer';
import Header from '../components/Header';

describe('Renders Header', () => {
  it('renders', () => {
    const tree = renderer
      .create(
        <Header />,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
