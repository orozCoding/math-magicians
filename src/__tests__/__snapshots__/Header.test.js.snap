// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Header renders 1`] = `
<header
  className="d-flex"
  id="header"
>
  <div
    className="logo-container d-flex"
  >
    <p
      className="logo"
    >
      Math Magicians
    </p>
  </div>
  <div>
    <nav
      className="navBar"
    >
      <ul
        className="d-flex"
      >
        <li>
          <a
            aria-current="page"
            className="activeLink"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li>
          <a
            className="none"
            href="/Calculator"
            onClick={[Function]}
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            className="none"
            href="/Quote"
            onClick={[Function]}
          >
            Quote
          </a>
        </li>
      </ul>
    </nav>
  </div>
</header>
`;
